Update signup.php

Samrusike opened the pull request:
For the LRR website, we don't need to display both the sign-in and sign-up forms on one page. The best thing that we can do to have a clean and neat LRR website UI/UX we should only display the logo and sign in form. Then we are also going to put a signup link on the sign-in page that will lead to the signup form and on the signup form the top left corner lrr logo will lead us back to the sign-in form. 
On the sign-up form I have removed the portal for inputting the passport and added an if statement for inputting student number on the same form.

Samrusike committed:
Update signup.php

For the LRR website, we don't need to display both the sign-in and sign-up forms on one page. The best thing that we can do to have a clean and neat LRR website UI/UX we should only display the logo and sign in form. Then we are also going to put a signup link on the sign-in page that will lead to the signup form and on the signup form the top left corner lrr logo will lead us back to the sign-in form. 
On the sign-up form I have removed the portal for inputting the passport and added an if statement for inputting student number on the same form.

enockkays committed:
Update index.php

Updated the Logo from "LRRS" to "LRR".
Removed the signup form from the index.php file (Index.php file had two forms now with only one form for signing in and a link that leads to the signup form).
On the signup form, we have removed the portal for putting the passport because I have realized that our colleagues from this repository "Ashik-Nafis_Bug 53" only removed the passport portal from the form. 
We also added a background and some CSS properties for the hovers effects on the new form.

Samrusike committed:
Add files via upload

This is the new background image for the lrr sign in form.

enockkays committed:
Update recover_password.php

We added first increased the padding for the recover button and also added some CSS properties for the hover effect.

Samrusike committed:
Update Script.php

Line 44, 53 - 60  we removed the passport portal not just from the form as our colleagues did but completely. Apart from the change above we also added a portal for inputting the student number on line 64.

enockkays committed:
Update Script.php

Student can create the accounts on their own without the need for the Teacher to create the accounts for the students. 
Also removed the inputs for the passport.

Samrusike committed:
Update Script.php

When a student enters a student number without the email he/she can not recover their password unless the enter both details and if not all details are correct the following error messages will be displayed; "Invalid student number" or "Invalid email address". These error messages will be displayed on the same page without navigating to the next page as before.
Line 222 - 224 and 230 - 232.

On Line 246 - 247 after resetting your password the recover buttons navigates direct to the signup.php where you can create a new account.

enockkays committed:
Update Script.php

Removed the passport number entries from the code.

enockkays committed:
Update Header.php

Removed the letter S from the "LRRS" to "LRR".

spm2020spring commented:
@enockkays @Samrusike 

Could you sync with the master branch before you push your branch?

Could you show me a front page without the background image (since it may take a while to load the image when people have a poor Internet connection)?

-Hui

Samrusike committed:
Update index.php

I  have removed the background and the "S" in the name on the footer too make it consistent  with the rest as we renamed it to LRR only.

enockkays commented:
The current UI looks better and clean, even though not the best and it can still be improved.
